manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -19,61 +19,140 @@
+    section("Scan hints") {
+      p("""If your query matches large parts of an index, it might be faster to scan the label and filter out nodes that do not match.
+          |To do this, you can use `USING SCAN variable:Label` after the applicable `MATCH` clause.
+          |This will force Cypher to not use an index that could have been used, and instead do a label scan.""")
+      section("Hinting a label scan") {
+        p("""If the best performance is to be had by scanning all nodes in a label and then filtering on that set, use `USING SCAN`.""")
+        query(s"""MATCH (s:Scientist)
+                 |USING SCAN s:Scientist
+                 |WHERE s.born < 1939
+                 |RETURN s.born AS $columnName""",
+              assertIntegersReturned(1938, 1867)) {
+          p("Returns all scientists born before 1939.")
+          profileExecutionPlan()
+        }
+      }
+    }
+    section("Join hints") {
+      p("""Join hints are the most advanced type of hints, and are not used to find starting points for the
+          |query execution plan, but to enforce that joins are made at specified points. This implies that there
+          |has to be more than one starting point (leaf), in order for the query to be able to join the two branches ascending
+          |from these leaves. Due to this nature, joins, and subsequently join hints, do have the potential to force
+          |the planner to pick a very bad starting point, which will negatively affect query performance. In other cases,
+          |the hint might force the planner to pick a _seemingly_ bad starting point, which in reality is a very good one.""")
+      section("Hinting a join on a single node") {
+        p("""In the example above using multiple index hints, we saw that the planner chose to do a join on the `cs` node.
+            |This means that the relationship between `wing` and `cs` was traversed in the outgoing direction, which is better
+            |statistically because the pattern `()-[:RESEARCHED]->(:Science)` is more common than the pattern `(:Scientist)-[:RESEARCHED]->()`.
+            |However, in the actual graph, the `cs` node only has two such relationships, so expanding from it will be beneficial
+            |to expanding from the `wing` node. We can force the join to happen on `wing` instead with a join hint.""")
+        query(s"""$matchString
+              |USING INDEX liskov:Scientist(name)
+              |USING INDEX conway:Scientist(name)
+              |USING JOIN ON wing
+              |RETURN wing.born AS $columnName""",
+              assertIntegersReturnedAndUsingHashJoin(1956)) {
+          p("Returns the birth date of Jeanette Wing, using a slightly better plan.")
+          profileExecutionPlan()
+        }
+      }
+      section("Hinting a join on multiple nodes") {
+        p("Force the query planner to produce a join between several specific points. This requires the query to expand from the same node from several directions.")
```

Comment:
"Force the query planner to produce a join between several specific points." -> 
"The query planner can be made to produce a join between several specific points".